core/interfaces.go

```diff
@@ -28,3 +28,9 @@ type Shared interface {
 	isEventAdmin(orgID string, appID string, eventID string, userID string, externalIDs map[string]string) (bool, error)
 	hasAttendedEvent(orgID string, appID string, eventID string, userID string, externalIDs map[string]string) (bool, error)
 }
+
+// Core exposes Core APIs for the driver adapters
+type Core interface {
+	RetrieveCoreUserAccountByCriteria(accountCriteria map[string]interface{}, appID *string, orgID *string) ([]model.CoreAccount, error)
```

Comment:
I fixed the same for Polls BB. No need to add code which is not used. Please look at Polls BB for reference.